packages/schemas/tables/sign_in_experiences.sql

```diff
@@ -1,10 +1,15 @@
+create type sign_in_method_state as enum ('primary', 'secondary', 'disabled');
+
 create table sign_in_experiences (
   id varchar(128) not null,
-  company_info jsonb /* @use CompanyInfo */ not null,
-  branding jsonb /* @use Branding */ not null,
-  terms_of_use jsonb /* @use TermsOfUse */ not null,
-  forget_password_enabled boolean not null default(true),
-  localization jsonb /* @use Localization */ not null,
-  sign_in_methods jsonb /* @use SignInMethodSettings */ not null,
+  branding jsonb /* @use Branding */ not null default '{}'::jsonb,
+  language_info jsonb /* @use LanguageInfo */ not null default '{}'::jsonb,
+  terms_of_use jsonb /* @use TermsOfUse */ not null default '{}'::jsonb,
+  forget_password_enabled boolean not null default false,
+  username_sign_in_state sign_in_method_state not default 'primary',
```

Comment:
why not simply give a required primary_sign_in_method and optional secondary_sign_in_methods field.

Reply:
The implementation follows the offline discussion conclusion, @IceHe would you mind updating the conclusion on Notion doc so that everybody can get the context?

Reply:
Temporarily keep it for now. (Need further exploration)
See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1646968789194879) on Slack.

Reply:
Updated related Notion [doc](https://www.notion.so/silverhand/Sign-in-experience-Storage-35410c0c9eff4c47950d0b31c0e5d25c).

Reply:
not null?